src/utils/index.ts

```diff
@@ -91,88 +83,72 @@ interface FindNewSecurityTokenParams extends FindEventParams {
   eventName: SecurityTokenRegistryEvents.NewSecurityToken;
 }
 
-interface FindCheckpointCreatedParams_3_0_0 extends FindEventParams {
-  eventName: SecurityTokenEvents_3_0_0.CheckpointCreated;
-}
-
-interface FindModuleAddedParams_3_0_0 extends FindEventParams {
-  eventName: SecurityTokenEvents_3_0_0.ModuleAdded;
-}
-
-interface FindControllerTransferParams_3_0_0 extends FindEventParams {
-  eventName: SecurityTokenEvents_3_0_0.ControllerTransfer;
-}
-
-interface FindErc20DividendDepositedParams_3_0_0 extends FindEventParams {
-  eventName: ERC20DividendCheckpointEvents_3_0_0.ERC20DividendDeposited;
+interface FindCheckpointCreatedParams extends FindEventParams {
+  eventName: SecurityTokenEvents.CheckpointCreated;
 }
 
-interface FindEtherDividendDepositedParams_3_0_0 extends FindEventParams {
-  eventName: EtherDividendCheckpointEvents_3_0_0.EtherDividendDeposited;
+interface FindModuleAddedParams extends FindEventParams {
+  eventName: SecurityTokenEvents.ModuleAdded;
 }
 
-interface FindModifyKycDataParams_3_0_0 extends FindEventParams {
-  eventName: GeneralTransferManagerEvents_3_0_0.ModifyKYCData;
+interface FindControllerTransferParams extends FindEventParams {
+  eventName: SecurityTokenEvents.ControllerTransfer;
 }
 
-interface FindModifyInvestorFlagParams_3_0_0 extends FindEventParams {
-  eventName: GeneralTransferManagerEvents_3_0_0.ModifyInvestorFlag;
+interface FindErc20DividendDepositedParams extends FindEventParams {
+  eventName: ERC20DividendCheckpointEvents.ERC20DividendDeposited;
 }
 
-interface FindModifyKycDataParams_3_1_0 extends FindEventParams {
-  eventName: GeneralTransferManagerEvents_3_1_0.ModifyKYCData;
+interface FindEtherDividendDepositedParams extends FindEventParams {
+  eventName: EtherDividendCheckpointEvents.EtherDividendDeposited;
 }
 
-interface FindModifyInvestorFlagParams_3_1_0 extends FindEventParams {
-  eventName: GeneralTransferManagerEvents_3_1_0.ModifyInvestorFlag;
+interface FindModifyKycDataParams extends FindEventParams {
+  eventName: typeof GeneralTransferManagerEvents.ModifyKYCData;
```

Comment:
This was necessary because the way these events are being exported makes it so the enum values can't be used directly as types, only as values (hence the need for `typeof`)